src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -347,4 +348,19 @@ private CallbackResponse validateLengthOfUnemployment(CallbackParams callbackPar
             .errors(errors)
             .build();
     }
+
+    private CallbackResponse repaymentPlan(CallbackParams callbackParams) {
```

Comment:
Please add test for this method in line with other callback methods.

Reply:
test cases already created